collector/xml-eater.py

```diff
@@ -0,0 +1,53 @@
+from lxml import etree 
+import numpy as np
+import re
+def xml_eater(xml_file, sample_rate, deliminator, discard=None):
+	"""	xml_file: file to be parsed
+	   	sample_rate: sample rate of audio that goes with xml_file
+		deliminator: deliminator for speaker labels.
+             	e.g. if we have "[M] Something something talking blah blah", 
+                  then deliminator is a regex "\[[A-Z]\]". (match open square bracket, any single letter, close square bracket.)
+		discard: discard any lines that contain this regex (e.g., if they label sound effects in a consistant way, we can get rid of 
+		those pretty easily using this param.)"""                      
+	# open xml
+	xml_file = open(xml_file).read()
+	tree = etree.fromstring(xml_file)	
+	starts = tree.xpath("text/@start")
+	texts  = [x.text.strip("\n") for x in tree.xpath("text")]
+	durs   = tree.xpath("text/@dur") 
+	speakers = [] # list of all possible speakers, used to determine size and entries of np array
+	# fill speakers
+	for text in texts:
+		for speaker in re.findall(deliminator, text):
+			if speaker not in speakers: speakers.append(speaker)
```

Comment:
For stuff like this, can you move the body of the `if` statement to the next line?